### DISCUSSION TITLE
Why convert image using Magick lib is brighter than generated image by Canon camera?

### DISCUSSION THREAD

**Discussion by ilho87** [dlemstra/Magick.NET#1291] [Help] [answered]
If there is similar discussions, I'm sorry.
I could not found any similar discussion in this Discussions menu.

By referring to the sample code, I converted the CR2 to an image in jpg format.
`var image = new MagickImage(filePath);`
`image.Write("testjpgimage.jpg");`
Of course it was a success.
![captureImage](https://user-images.githubusercontent.com/13030769/199202972-c178d954-7bf8-4ef3-87c6-b3a3cf778ce4.JPG)

(IMG_3926.JPG is a jpg automatically generated by Canon camera when taking a picture, and testjpgimage.jpg is an image converted to jpg by CR2.)

But I wonder why convert image using Magick lib is brighter than generated image by Canon camera.
How do I make the converted image look like a Canon camera generated image?

I would be grateful for any advice!

**Comment by dlemstra** [comment, marked as answer]
You probably need to tweak the `DngReadDefines` and specify this in the `readSettings` when you create the image.

**Reply by ilho87** [reply]
Thank you for your reply.

I will try it what you let me know.
